packages/manager-react-components/src/components/content/headers/headers.component.tsx

```diff
@@ -1,12 +1,12 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
 import { OdsText } from '@ovhcloud/ods-components/react';
 import { Title, Subtitle } from '../../typography';
 
 export interface HeadersProps {
-  title?: string;
-  subtitle?: string;
-  description?: string;
-  headerButton?: React.ReactElement;
+  title?: ReactNode;
+  subtitle?: ReactNode;
+  description?: ReactNode;
```

Comment:
Lets discuss with DEsign team if we need to make `ReactNode` for title, subtitle & decsription.

Reply:
any update about this @anooparveti ?